plugins/woocommerce/src/Internal/Admin/Loader.php

```diff
@@ -445,20 +444,19 @@ public static function add_component_settings( $settings ) {
 	 *
 	 * @param array $statuses Order statuses.
 	 * @return array formatted statuses.
+	 *
+	 * @deprecated migrate to \Automattic\WooCommerce\Internal\Admin\Settings instead.
```

Comment:
Since we're deprecating this should we also add a `wc_deprecated_function` in this function and change the use of `self::get_order_statuses()` in this class?

Reply:
> and change the use of self::get_order_statuses() in this class

Addressed in f713df1070e1357988ced54c7a186b1d71d05beb

Reply:
> Since we're deprecating this should we also add a wc_deprecated_function in this function

Addressed in 52d4c175df66d99f239337f8721b44766819ac77